src/installation/step-by-step.rst

```diff
@@ -228,8 +228,8 @@ If you have an GBFS feed, you can add an otp updater config to ``router-config.j
     ...
 
 The building of the graph happens with the `mfdz OpenTripPlanner fork <https://github.com/mfdz/opentripplanner>`__.
-It is important that the OpenTripPlanner version that builds the graph is the same that later serves the graph. If you want to update, get the latest docker image tag from the
-`docker hub page of mfdz/opentripplanner <https://hub.docker.com/r/mfdz/opentripplanner/tags>`__ and modify ``OTP_VERSION`` in the Dockerfile.
+It is important that the OpenTripPlanner version that builds the graph is the same that later serves the graph. If you want to update, get the latest docker image tag from the Docker Hub page of
+'mfdz/opentripplanner <https://hub.docker.com/r/mfdz/opentripplanner/tags>`__ and modify ``OTP_VERSION`` in the Dockerfile.
```

Comment:
This this change intentional? What do you think about this formatting?

```md
It is important that the OpenTripPlanner version that builds the graph is the same that later serves the graph. If you want to update, get the latest docker image tag from the [`mfdz/opentripplanner` Docker Hub page](https://hub.docker.com/r/mfdz/opentripplanner/tags) and modify `OTP_VERSION` in the Dockerfile.
```

Reply:
Hi @derhuerst ,
I am always feeling comfortable writing and reading markdown. To be honest I do not even know the `rst` file extension and the respective text format yet.

I agree to your suggestion. What is the general procedure to go on with this PR? Shall  I integrate your suggestion and make an additional commit?

Reply:
Oh, i totally missed that we're not looking a Markdown file. Don't mind my comment then.